hilti/toolchain/include/ast/attribute.h

```diff
@@ -68,6 +119,22 @@ class Attribute : public Node {
      */
     Result<bool> coerceValueTo(Builder* builder, QualifiedType* dst);
 
+    static bool isOneOf(Kind kind, const std::unordered_set<Kind>& kinds) { return kinds.count(kind) > 0; }
```

Comment:
nit: all users seem to be fine with passing a `std::initializer_list` (by value) which should be cheaper to construct.